src/components/app/app.tsx

```diff
@@ -1,13 +1,46 @@
+import {Route, BrowserRouter, Routes} from 'react-router-dom';
+import {HelmetProvider} from 'react-helmet-async';
+import {PlacesFoundCountType, AppRoute, AuthorizationStatus} from '../../constants';
 import MainPage from '../../pages/main-page/main-page';
+import OfferPage from '../../pages/offer-page/offer-page';
+import FavoritesPage from '../../pages/favorites-page/favorites-page';
+import LoginPage from '../../pages/login-page/login-page';
+import PrivateRoute from '../private-route/private-route';
+import NotFoundPage from '../../pages/not-found-page/not-found-page';
```

Comment:
Если хочешь поставь плагин vite ts config paths и в tsconfig пропиши алиас '@/' для src тогда можно будет импортировать так 
```suggestion
import NotFoundPage from '@/pages/not-found-page/not-found-page';
```